R/civis_ml_utils.R

```diff
@@ -193,3 +193,41 @@ is_multiclass <- function(model) {
 is_multitarget <- function(model) {
   length(get_model_data(model, "n_unique_targets")) > 1
 }
+
+#' coef
+#' @details Outputs coefficients with names in the style of `stats::coef`
+#' @param model civis_ml_classifier
+#' @return coefs a vector of coefficients or `NULL` if the classifier does not have coefficients
+#' @export
+coef <- function(model) {
+  UseMethod("coef")
+}
+
+#' Function to get coefficients from a civis_ml_classifier
+#' @details Outputs coefficients with names in the style of `stats::coef`
+#' @param model civis_ml_classifier
+#' @return coefs a vector of coefficients or `NULL` if the classifier does not have coefficients
+#' @export
+coef.civis_ml <- function(model) {
+
+  # fields needed aren't present (i.e.: tree-based model)
+  if(is.null(model[["model_info"]][["model"]][["parameters"]][["coef"]])) {
+    return(NULL)
+  }
```

Comment:
`} else {`

Reply:
I believe I've fixed the issue with re-implementing the generic (see screenshot).  

![screen shot 2018-08-22 at 14 26 54](https://user-images.githubusercontent.com/18688162/44485843-93dd1b80-a617-11e8-91e2-73a900c6bd53.png)

When I do this, the 'export(coef)' line In NAMESPACE goes away but the one you highlighted 'S3method(coef,civis_ml)' does not -- is that ok?